pkgs/development/python-modules/fastecdsa/default.nix

```diff
@@ -3,21 +3,28 @@
 , fetchPypi
 , gmp
 , pytestCheckHook
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "fastecdsa";
-  version = "2.2.2";
+  version = "2.2.3";
   format = "setuptools";
 
+  disabled = pythonOlder "3.6";
+
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1eb6f3ac86ec483a10df62fcda1fb9a9d5d895a436871a8aa935dd20ccd82c6f";
+    sha256 = "sha256-JpvbD2GLOPj2rsnSPSPbUYBGw87gGpVPpqpzIqGn248=";
```

Comment:
I'd rather keep hex notation of the hash so it could be trivially checked on PyPI.

Reply:
Sure, I will update it.